pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -65,6 +66,25 @@ servingInfo:
   bindAddress: https://0.0.0.0:8443
   certFile: /var/serving-cert/tls.crt
   keyFile: /var/serving-cert/tls.key
+`
```

Comment:
How do you feel about renaming the mocks to express the differences?  I found that I was jumping back and forth to remember what the input actually was, and if the output was what I expected.  Meaning:

```go
// inputs, to give to test funcs
managedCMOnline := `kind: ConsoleConfig...`
// outputs, to test against
finalCMDefaultOKD := `kind: ConsoleConfig...`
finalCMOnline := `kind: ConsoleConfig...`
finalCMDedicated := `kind: ConsoleConfig...`

// lets the test look like this:
args: args{
    managedConfig: &corev1.ConfigMap{
        // i can see managed provides an online brand
        Data: map[string]string{configKey: managedCMOnline},
    },
    // other configs....
    // operator config provides DEDICATED brand
},
want: &corev1.ConfigMap{
    ObjectMeta: metav1.ObjectMeta{
        Name:        api.OpenShiftConsoleConfigMapName,
        Namespace:   api.OpenShiftConsoleNamespace,
        Labels:      map[string]string{"app": api.OpenShiftConsoleName},
        Annotations: map[string]string{},
    },
    // FINALLY I expect final output, dedicated, to win, because operator config overrides managed
    Data: map[string]string{configKey: finalCMDedicated},
},			
```